x/vm/wrappers/bank_test.go

```diff
@@ -309,19 +286,18 @@ func TestSendCoinsFromModuleToAccount(t *testing.T) {
 			},
 		},
 		{
-			name:        "success - convert 18 decimals amount to 6 decimals with single token",
-			evmDenom:    TokenDenom,
-			evmDecimals: 6,
+			name:     "success - convert 18 decimals amount to 6 decimals with single token",
```

Comment:
same here - seems like we're no longer converting

Reply:
[ditto](https://github.com/cosmos/evm/pull/69#discussion_r2068352936)